src/core/references/index.ts

```diff
@@ -150,55 +149,75 @@ export const WETH_BLAST_ADDRESS = '0x4300000000000000000000000000000000000004';
 export const USDB_BLAST_ADDRESS = '0x4300000000000000000000000000000000000003';
 
 export const NATIVE_ASSETS_PER_CHAIN: Record<ChainId, AddressOrEth> = {
-  [ChainId.mainnet]: ETH_ADDRESS as Address,
-  [ChainId.hardhat]: AddressZero as Address,
-  [ChainId.sepolia]: AddressZero as Address,
-  [ChainId.holesky]: AddressZero as Address,
   [ChainId.arbitrum]: ETH_ARBITRUM_ADDRESS as Address,
   [ChainId.arbitrumSepolia]: AddressZero as Address,
+  [ChainId.avalanche]: AVAX_AVALANCHE_ADDRESS as Address,
+  [ChainId.avalancheFuji]: AddressZero as Address,
+  [ChainId.base]: ETH_BASE_ADDRESS as Address,
+  [ChainId.baseSepolia]: AddressZero as Address,
+  [ChainId.blast]: AddressZero as Address,
+  [ChainId.blastSepolia]: AddressZero as Address,
```

Comment:
isn't it better to directly type `AddressZero` as `Address` instead of casting every time we use it?